part1-장재혁-week4/src/signin.js

```diff
@@ -0,0 +1,77 @@
+
+const emailInput = document.getElementById('emailInput');
+const passwordInput = document.getElementById('passwordInput');
+const ctaSignIn = document.getElementById('ctaSignIn');
+let errorEmail = document.getElementById('errorEmail');
+let errorPassword = document.getElementById('errorPassword');
+
+let eyeButtons = document.getElementsByClassName('eye-button');
+let eyeOffs = document.getElementsByClassName('eyeOff');
+
+
+const emailCheck = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+emailValidChecker =(email) => {
+    if(emailCheck.test(email) === false) return false;
+    else return true;

```

Comment:
한 줄로 축약할 수 있을 거 같아요!
```suggestion
    return emailCheck.test(email)
```